data/supergroups.yml

```diff
@@ -27,23 +27,21 @@ content_purpose_supergroup:
         - research_and_data
 
 documentation : |
-  ##Content Purpose Supergroups
-
   These are the grouping that the navigation team came up with
   as a result of an investigation done using the "Jobs to be Done" framework in Q4.
 
-  There are 5 high level Content Purpose Supergroups which each group together some subgroups
-  which in turn group `document_types`.
+  There are 5 high level Content Purpose Supergroups which each group together some subgroups.
```

Comment:
For consistency, I'd say we should refer to subgroups here as 'Content Purpose Subgroups' to make things clearer and also match with our use of 'Content Purpose Supergroups'

Reply:
I agree with @vanitabarrett. Once that change is made, I'm happy for this to be merged.